src/CSI.IBTA.Employer/Clients/SettingsClient.cs

```diff
@@ -0,0 +1,43 @@
+﻿using CSI.IBTA.Employer.Endpoints;
+using CSI.IBTA.Employer.Interfaces;
+using CSI.IBTA.Shared.DTOs;
+using Newtonsoft.Json;
+using System.Net.Http;
+
+namespace CSI.IBTA.Employer.Clients
+{
+    internal class SettingsClient : ISettingsClient
+    {
+        private readonly AuthorizedHttpClient _httpClient;
+
+        public SettingsClient(AuthorizedHttpClient httpClient)
+        {
+            _httpClient = httpClient;
+            _httpClient.SetBaseAddress("UserServiceApiUrl");
+        }
+
+        public async Task<GenericResponse<SettingsWithEmployerStateDto>> GetClaimSetting(
+            int employerId)
+        {
+            var requestUrl = string.Format(EmployerEndpoints.GetEmployerSetting, employerId, "ClaimFilling");
+            var response = await _httpClient.GetAsync(requestUrl);
+            var responseContent = await response.Content.ReadAsStringAsync();
```

Comment:
content could be deserialized with ReadFromJsonAsync. 

![image](https://github.com/mykytatishkin/csi-training-healthcare-2024/assets/2743649/8692607a-772d-4a87-a608-56bd4a0e46fd)

https://gist.github.com/AnthonyGiretti/bc8b6d8a4f4d6229c9de9510de4c2236